doc/Release_Build_Creation.rst

```diff
@@ -8,19 +8,35 @@ This file consists of the procedure that should be followed to generate the rele
 Prerequisites to setup Virtual Machine
 *****************************************
 
-- Single-Node VM deployment:
+- **Single-Node VM deployment:**
   
   - Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system with the following configuration in Virtual Machine (VM):
-  
-    - RAM: 4GB
-    - CPU: 1 core
-    - NIC: 3
-    - Total Disks: 3
+  - Create VM(s) with at least 4 vCPUs and 4GB of RAM.
+  - Minimum 3 NIC is required on different network as per recommendation
+  - Storage Configuration:
     
-      - Data Disk: 1 (Capacity: 50GB)
-      - Metadata Disk: 1 (Capacity: 10% of total Data Disk Size)
+      - Usecase-1:
+      
+        - Min 9 Disks
+        
+          - Data Disks - Min 8 Disks (Capacity 10GB+)
+          - Metadata Disks - Min 1 Disks (Capacity - 5GB+)
+      
+      - Usecase-2:
```

Comment:
Is this use case is about EC as 1+0+0, if so,
Erasure coding with units N(data) + K (parity) + S (spare) as 1 + 0 + 0.

Reply:
done